dm/master/server.go

```diff
@@ -1838,9 +1838,16 @@ func (s *Server) listMemberMaster(ctx context.Context, names []string) (*pb.Memb
 
 	etcdMembers := memberList.Members
 	masters := make([]*pb.MasterInfo, 0, len(etcdMembers))
-	client := http.Client{
-		Timeout: 1 * time.Second,
+
+	client := &http.Client{}
+	if len(s.cfg.SSLCA) != 0 {
+		inner, err := toolutils.ToTLSConfigWithVerify(s.cfg.SSLCA, s.cfg.SSLCert, s.cfg.SSLKey, s.cfg.CertAllowedCN)
```

Comment:
Does http client need `CertAllowedCN`? Is `ToTLSConfig` enough?

Reply:
If user set a wrong common name, we can verify it in http client. We also do it when master create etcd grpc client.
Do you think we don't need to verify the common name for http? It's enough though.